### DISCUSSION TITLE
client parameters

### DISCUSSION THREAD

**Discussion by wofo99** [fedorovvl/tso_client#343] [Q&A]
seems I cannot run 

client.exe --login myname@gmail.com --password mypw --lang de

It does not fill in the language.. Any idea?

**Comment by Cikasz** [comment]
I run game a browser and login ubisoft account. 

If I enter my ubisoft data in the EXE, I get this error message:
Login attempt #0
Authentication ubi
Successful authentication 
Authentication uplay
E-mail or password not correct.

But the data is good because I use it to enter the ubi client!

**Reply by Cdrwbw** [reply]
Do you have dual authentication activated on the ubisoft account? If so, deactivate dual authentication.

**Reply by Cikasz** [reply]
Yes i linked the google authenticator

**Reply by skelgaard** [reply]
that doesn't work on client

**Reply by Cikasz** [reply]
thanks i try

**Comment by wofo99** [comment]
My solution: I changed the script so it does not move Settings.dat to %uname.dat but just copies it. At first run I have to give Country info, but it is kept for all subsequent runs. Used to activate account(s) on different hosts when travelling with DSO on stick...
